css-canvas-medical/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container padding-top-bottom-15">
+    <div class="row">
+      <div class="column-half">
+        <img src="images/logo-blue.svg" alt="logo-blue">
+      </div>
+      <div class="column-half flex-end">
+        <div class="nav-bar">
+          <a>Product</a>
+          <a>Implementation</a>
+          <a>About Us</a>
+          <a>Blog</a>
+        </div>
+        <div class="contact-us-button">
+          <button>Contact Us</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="border-bottom margin-bottom-75"></div>
+  <div class="container margin-bottom-75">
```

Comment:
You shouldn't need to make an individual `container` for each section. In general, you will have one `container` that holds all the `row` elements necessary for your application.